src/main/java/com/stripe/param/SubscriptionItemUpdateParams.java

```diff
@@ -72,6 +73,14 @@
   @SerializedName("plan")
   Object plan;
 
+  /** The ID of the price object. */
+  @SerializedName("price")
+  Object price;
```

Comment:
Same as for InvoiceItem, I don't understand why we would let you unset a `Price`